pkg/agent/plugin/workloadattestor/windows/windows_posix.go

```diff
@@ -4,38 +4,17 @@
 package windows
 
 import (
-	"context"
-
 	workloadattestorv1 "github.com/spiffe/spire-plugin-sdk/proto/spire/plugin/agent/workloadattestor/v1"
-	configv1 "github.com/spiffe/spire-plugin-sdk/proto/spire/service/common/config/v1"
 	"github.com/spiffe/spire/pkg/common/catalog"
-	"google.golang.org/grpc/codes"
-	"google.golang.org/grpc/status"
-)
-
-var (
-	errNotSupported = "plugin not supported in this platform"
 )
 
 type Plugin struct {
-	workloadattestorv1.UnsafeWorkloadAttestorServer
-	configv1.UnsafeConfigServer
+	workloadattestorv1.UnimplementedWorkloadAttestorServer
 }
 
 func builtin(p *Plugin) catalog.BuiltIn {
-	return catalog.MakeBuiltIn(pluginName,
-		workloadattestorv1.WorkloadAttestorPluginServer(p),
-		configv1.ConfigServiceServer(p),
-	)
+	return catalog.MakeBuiltIn(pluginName, workloadattestorv1.WorkloadAttestorPluginServer(p))
 }
 func New() *Plugin {
 	return &Plugin{}
 }
-
-func (p *Plugin) Attest(ctx context.Context, req *workloadattestorv1.AttestRequest) (*workloadattestorv1.AttestResponse, error) {
-	return nil, status.Error(codes.Unimplemented, errNotSupported)
-}
-
-func (p *Plugin) Configure(ctx context.Context, req *configv1.ConfigureRequest) (*configv1.ConfigureResponse, error) {
```

Comment:
I think we still need the config service implementation so we can return not implemented for posix. Plugins aren't required to have a config service so if we just leave this out, the plugin will load successfully and just fail to attest any workloads.